<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="Signup">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border: none;
background-color: none;
font: 75 28pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
padding bottom: 7px;</string>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: rgb(255, 255, 255);
border-bottom: 2px solid rgb(255, 35, 141);
font: 11pt &quot;Cambria&quot;;
color: rgb(0, 0, 0);
padding bottom: 7px;</string>
   </property>
   <property name="placeholderText">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: rgb(255, 255, 255);
border-bottom: 2px solid rgb(255, 35, 141);
font: 11pt &quot;Cambria&quot;;
color: rgb(0, 0, 0);
padding bottom: 7px;</string>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signupbutton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>500</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>20</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;Comic Sans MS&quot;;
font: 75 20pt &quot;Calibri&quot;;
color: rgb(255, 255, 255);
background-color: rgb(200, 151, 67);</string>
   </property>
   <property name="text">
    <string>Signup</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>580</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color:none;</string>
   </property>
   <property name="text">
    <string>Already have an account?</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmpass">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>340</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: rgb(255, 255, 255);
border-bottom: 2px solid rgb(255, 35, 141);
font: 11pt &quot;Cambria&quot;;
color: rgb(0, 0, 0);
padding bottom: 7px;</string>
   </property>
   <property name="placeholderText">
    <string>Confirm Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signupbutton_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(200, 151, 67);</string>
   </property>
   <property name="text">
    <string>Signup</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginbuttons">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>570</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: none;
color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>Log In</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="show_password_checkbox">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>270</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="show_confirmpass_checkbox">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>350</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>861</width>
     <height>631</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>background.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>Signup</zorder>
  <zorder>email</zorder>
  <zorder>password</zorder>
  <zorder>signupbutton</zorder>
  <zorder>label</zorder>
  <zorder>confirmpass</zorder>
  <zorder>signupbutton_2</zorder>
  <zorder>loginbuttons</zorder>
  <zorder>label_2</zorder>
  <zorder>show_password_checkbox</zorder>
  <zorder>show_confirmpass_checkbox</zorder>
 </widget>
 <tabstops>
  <tabstop>signupbutton_2</tabstop>
  <tabstop>signupbutton</tabstop>
  <tabstop>email</tabstop>
  <tabstop>password</tabstop>
  <tabstop>confirmpass</tabstop>
  <tabstop>loginbuttons</tabstop>
 </tabstops>
 <resources>
  <include location="test.qrc"/>
 </resources>
 <connections/>
</ui>
